handlers/load_test_meshes_handler.go

```diff
@@ -0,0 +1,35 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/layer5io/meshery/models"
+	SMP "github.com/layer5io/service-mesh-performance/spec"
+	"github.com/sirupsen/logrus"
+)
+
+// SMPMeshes defines the JSON payload structure for available meshes api
+type SMPMeshes struct {
+	AvailableMeshes []string `json:"available_meshes,omitempty"`
+}
+
+// GetSMPServiceMeshes handles the available meshes request
+func (h *Handler) GetSMPServiceMeshes(w http.ResponseWriter, r *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+	meshes := SMPMeshes{
+		AvailableMeshes: make([]string, 0, len(SMP.ServiceMeshes_Type_name)),
+	}
```

Comment:
Why is this logic needed? Can't we use 
```suggestion
	var meshes SMPMeshes
```

Reply:
This was just to set initial capacity of the slice or else while appending the elements to the slice, the capacity would double (albeit till a certain threshold) which could make it a bit slower. Here we already have the information of the maximum capacity for the slice that we may need so I just utilized it here :sweat_smile:

Reply:
This is a very minor optimization which makes code rather difficult to read. But still, let's move ahead with this.